manifests/init.pp

```diff
@@ -145,6 +145,7 @@
   $cacheupdatefrequency                     = $zabbix::params::server_cacheupdatefrequency,
   $startdbsyncers                           = $zabbix::params::server_startdbsyncers,
   $historycachesize                         = $zabbix::params::server_historycachesize,
+  $historyindexcachesize                    = $zabbix::params::server_historyindexcachesize,
```

Comment:
can you please add a dataype to the parameter?

Reply:
Done.